ballerina/block_stream.bal

```diff
@@ -18,14 +18,13 @@ import ballerina/jballerina.java;
 # The read-only byte array that is used to read the byte content from the streams.
 public type Block readonly & byte[];
 
-# The `BlockStream` is used to initialize a stream of type `Block`. This `BlockStream` refers to the stream that is embedded to
-# the I/O byte channels.
+# The BlockStream is a stream of type `Block` and it refers to the stream that is embedded to the I/O byte channels.
 public class BlockStream {
     private ReadableByteChannel readableByteChannel;
     private int blockSize;
     private boolean isClosed = false;
 
-    # Initialize a `BlockStream` using an `io:ReadableByteChannel`.
+    # Initializes a BlockStream.
```

Comment:
```suggestion
    # Initializes the `BlockStream` instance.
```

Reply:
Yes I agree and your suggestion LGTM. However my concern was whether we should mention Ballerina types with their module prefix, whenever we refer them in the documentation 

e.g: w.r.t. your latest suggestion, my concern in if it should be  ```Initializes a stream of `Block` objects.``` or ```Initializes a stream of `io:Block` objects.```

As I've mentioned above the reason is these docs are written for library users, and they always see (and are required to use) the types with the module prefix in the code, so it might be clearer to include it here as well. wdyt?